### ISSUE TITLE
Slides in slideshow element should not be navigable by TAB

### ISSUE THREAD

**Issue by mattlam-uw** [uweb/uw-2014#23]
# Description
This is in response to my comment on Issue https://github.com/uweb/uw-2014/issues/13. 

Looking at other keyboard navigation and accessibility patterns on the web, we have determined that individual slides in a slideshow element should not have their own tab stops. Rather, when the slideshow element itself has focus, keyboard navigating users should be able to navigate the slides using arrow keys. Hitting TAB when the slideshow element already has focus, should take focus to the next element on the page.

# Steps to Reproduce
1. Open any commonly-used browser (IE, Firefox, Chrome) and navigate to a page hosting the slideshow component (e.g. http://www.washington.edu/research/)
2. Use TAB to navigate to the point where the slideshow component has just received focus
3. Hit TAB or SHIFT-TAB one more time

# Expected Result
* __Case: TAB__: the next element on the page (the one after the slide show) receives focus
* __Case: SHIFT-TAB__: the previous element on the page (the one prior to the slide show) receives focus

# Actual Result
With both TAB and SHIFT-TAB, the slideshow will advance to the next slide

**Comment by nwinkelbauer** [comment]
Issue addressed by Commit 876b2d2
